wolfssl/version.h.in

```diff
@@ -23,14 +23,22 @@
 #ifndef WOLFSSL_VERSION_H
 #define WOLFSSL_VERSION_H
 
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#ifdef HAVE_CONFIG_H
```

Comment:
These config.h and settings.h go in the .c, not the .h.

Reply:
I added the `config.h` and `settings.h` using the model from [ssl.h](https://github.com/wolfSSL/wolfssl/blob/b8e61a241be97e84ebcb8128340203801fa92b39/wolfssl/ssl.h#L33); in particular I needed to include `visibility.h` in order to use the [WOLFSSL_API](https://github.com/gojimmypi/wolfssl/blob/aeebb9fef6a0146ff5336439fac2238ef2550041/wolfssl/version.h#L39) macro, without which, the linker could not find `ShowExtendedSystemInfo()`.  Is there a preferred way of doing this?